studio/components/to-be-cleaned/Charts/ChartRenderer.tsx

```diff
@@ -179,7 +179,7 @@ export function BarChart({
       <div style={{ width: '100%', height: '160px' }}>
         {hasData ? (
           <>
-            <ResponsiveContainer>
+            <ResponsiveContainer width='100%' height={160}>
```

Comment:
we can set both height and width to 100%, that way the div wrapper handles the fixed height instead of us having to set the value twice.

Reply:
When setting the height to 100% the warning happens when I run the test locally.

Reply:
Have a look at the rechart src: 
https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.tsx#L158

Specifically the `getContainerSizes()`, the `renderChart`, and the `return` of the component.

What is resulting in the warnings being emitted isn't so simple, so here's the full explanation:

Because on initial mount and subsequent resizes, the [`clientHeight`](https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.tsx#L61) [(MDN)](https://developer.mozilla.org/en-US/docs/Web/API/Element/clientWidth) of the component is used to calculate the `containerHeight`. Subsequently, to form the `calculatedHeight`, this `containerHeight` is then used if the [provided height is a percent string](https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.tsx#L102), otherwise it uses the fixed `height` prop value. This `calculatedHeight` height is then provided to the inner cloned child. 

Notice that if it is a percent, then the inner cloned element is set to clientHeight, then the percentage is passed to the [`<div>` element](https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.tsx#L162) in the `return`, resulting in the wrapper `<div>` doing the percentage scaling.    

If you see the [warning](https://github.com/recharts/recharts/blob/master/src/component/ResponsiveContainer.tsx#L120) (where you don't pass any props), you'll see that the calculatedHeight/calculatedWidth is both `0` when the console warning is emitted, meaning that the `clientHeight` is set to 0 on initial mount, which is causing the console warnings to be emitted on jsdom mount (if I were to hazard a guess, it is likely zero due to the rechart resizing callback not getting triggered in jsdom or clientHeight api  not being available)

Because of the implementation, it rules out a few possible fixes:
- we can't wrap the chart with a div that has 100% height + width, because they wont affect clientHeight
- we can't pass the minHeight prop, as it won't affect clientHeight but only the wrapper <div>
- mocking rechart's internal apis would be too much work, and likewise for stubbing jsdom's missing functionality.

So, instead of setting the fixed value in multiple places (which we want to avoid if possible, as there is another PR #4344 that changes the fixed height based on a prop), lets move the height constant to a render variable and then interpolate it to the style object height string. Also add in an inline comment that references this PR for future reference.

also, because of the warning logic used, we can omit the `width` and only provide the height prop.